pywren/wren.py

```diff
@@ -68,7 +70,9 @@ def remote_executor(config= None, job_max_runtime=3600,
     S3_PREFIX = config['s3']['pywren_prefix']
     invoker = invokers.SQSInvoker(AWS_REGION, SQS_QUEUE)
     return Executor(AWS_REGION, S3_BUCKET, S3_PREFIX, invoker, config,
-                    job_max_runtime, shard_runtime=shard_runtime)
+                    job_max_runtime)
+
+standalone_executor = remote_executor
```

Comment:
is this line necessary ? just checking as it wasn't in the original PR if i'm right